docs/scope.md

```diff
@@ -0,0 +1,53 @@
+# Document Purpose
+
+This document is a guide for contributors and Senate to decide if a feature is within "scope" for pokeemerald-expansion. If a feature is not in scope, then it should not be merged. Even if an opened PR is within scope, this does not mean it will be merged, as acceptance criteria will often come down to the details of the implementation.
+
+# Definitions
+
+* **Showdown Supported (SS)**: A core series game who's metagame can be played on Showdown.
+  * Notably, this is every [core series game](https://bulbapedia.bulbagarden.net/wiki/Core_series#List_of_core_series_games) except Pokémon Legends: Arceus and Pokémon Legends: Z-A.  
```

Comment:
We don't know yet how Legends Z-A will play or if it'll be supported by Showdown.